astroquery/hsa/tests/test_hsa_remote.py

```diff
@@ -0,0 +1,117 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import pytest
+
+import os
+
+from ..core import HSAClass
+from .test_hsa import TestHSA
+
+
+class TestHSARemote(TestHSA):
+
+    @pytest.mark.remote_data
+    def test_download_data_observation_pacs(self):
+        obs_id = "1342195355"
+        parameters = {'retrieval_type': "OBSERVATION",
+                      'observation_id': obs_id,
+                      'instrument_name': "PACS"}
+        expected_res = obs_id + ".tar"
+        hsa = HSAClass(self.get_dummy_tap_handler())
+        res = hsa.download_data(**parameters)
+        assert res == expected_res
```

Comment:
you may want to check the content of the downloaded data, too, here and in a few places below.

Reply:
Done.